### PR TITLE
chore: add workflow that pushes to Docker Hub on merge with main branch

### PR DESCRIPTION
**Pull request by mpro7**
I tested it w/o if condition and it pushed the image to DH.

### CODE REVIEW

File: .github/workflows/build-push.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Build and publish to Dockeer Hub
```
Comment (BalduinLandolt) [inline .github/workflows/build-push.yml:1]:
```suggestion
name: Build and publish to Docker Hub
```

File: .github/workflows/build-push.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      - name: Build and push image
+        run: sbt Docker/publish
+
```
Comment (BalduinLandolt) [inline .github/workflows/build-push.yml:32]: missing linebreak? github suggests so

**Review by BalduinLandolt — APPROVED** [review]
see my comments. Otherwise looking good, thx

File: .github/workflows/build-push.yml (hunk 2 above)
Comment (mpro7) [reply .github/workflows/build-push.yml]: Weird, it was there but maybe it didn't like the indentation. I pushed the fix.

**Pull request merged**